manim/mobject/types/vectorized_mobject.py

```diff
@@ -1388,6 +1388,22 @@ def set_location(self, new_loc):
 
 
 class CurvesAsSubmobjects(VGroup):
+    """Convert a curve's elements to submobjects.
+
+    Examples
+    --------
+    .. manim:: LineGradientExample
+        :save_last_frame:
+
+        class LineGradientExample(Scene):
+            def construct(self):
+                l = CurvedDoubleArrow(LEFT, RIGHT)
+                new_func = CurvesAsSubmobjects(l, background_stroke_width=0)
```

Comment:
Same comment as above, `l` should be avoided as a variable name.

Also, the example confuses me (or maybe I don't understand what `CurvesAsSubmobjects` is supposed to do: why are the arrow tips not included in the rendered image? (This is why I didn't suggest changing `l` to `arrow` or so.)

Reply:
![image](https://user-images.githubusercontent.com/44469195/102364734-117e7f80-3fb7-11eb-8d1d-b295bd78d583.png)
Oh, you are right, the CurvedDoubleArrow loses its tips when using CurvesAsSubmobjects.
I will change this to a parametric function